### DISCUSSION TITLE
plotly min error

### DISCUSSION THREAD

**Discussion by GiorgioMauro** [LayTec-AG/Plotly.Blazor#452] [General]
Hi everyone,

I’m still seeking help with an issue. Whenever I open a page with Plotly blazor, my application becomes unresponsive and everything freezes. Does anyone know how to fix this?

![image](https://github.com/user-attachments/assets/757795e9-8303-4935-b42b-2c87bd270526)

anyone know how?

thank y'all

**Comment by sean-mcl** [comment]
Are you trying to operate on the chart when it has not been rendered yet?
A minimal example would help.

There is an "AfterRender" action that you can use, to add the data when the chart has been rendered successfully.

Here you can see the example:

https://laytec-ag.github.io/Plotly.Blazor/

**Reply by GiorgioMauro** [reply]
No, I just load the data and that's all! The actions i'll do after the rendering. I'll paste some some:

![image](https://github.com/user-attachments/assets/301bd13b-1366-412a-bf12-4b626b8dc2ae)


and razor.cs

![image](https://github.com/user-attachments/assets/a2fb1011-505a-4fc3-a041-a6ec87d18b12)


and there are more method for the other action but i think you don't need it 'cause there are operation on click

**Comment by sean-mcl** [comment]
Do you still have Plotly.js referenced in Index.html? This is no longer necessary since 5.x.x, as the library is loaded lazy and async automatically. there could possibly be conflicts.

**Comment by GiorgioMauro** [comment]
I had in my _layout.cshtml but i just removed as you see. Maybe i have to refesh all the min.js or something?
![image](https://github.com/user-attachments/assets/525ad6e9-813c-4cca-9111-1d8e87af6a9f)
